setup.py

```diff
@@ -11,7 +11,7 @@
     author="Jacob Burch",
     author_email="jacobburch@gmail.com",
     url='https://github.com/jacobb/nap',
-    long_description=open('README.rst').read(),
+    long_description=open('README.rst', encoding='utf-8').read(),
```

Comment:
Has nap already dropped Python 2 support? If it hasn't, I think this approach will effectively drop it. See the discussion here: https://github.com/CashStar/nap/issues/20

Are we ok with dropping Python 2 support if we haven't already?

Reply:
I'm fine with taking the approach discussed in that thread to keep it backwards compatible, will update.